odoo/addons/base/ir/ir_attachment.py

```diff
@@ -438,6 +439,14 @@ def create(self, values):
         self.browse().check('write', values=values)
         return super(IrAttachment, self).create(values)
 
+    @api.one
+    def generate_access_token(self):
+        if self.access_token:
+            return self.access_token
```

Comment:
Seeing this, I think we should restrict the visibility of the `access_token` field to employees only, at the model level. There should be no need for portal/public users to view/modify the token. Either they received it or they didn't.

Reply:
I add the group 'base.group_user' on the access_token field